package.json

```diff
@@ -9,76 +9,78 @@
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
     "nx": "nx",
     "start": "nx serve",
-    "build": "nx run-many --target=build --configuration=production --all --parallel",
+    "build": "nx run-many --target=build --configuration=production --all",
```

Comment:
What issue force this?

Reply:
`ngcc` make CI fail when building multiple Angular applications in parallel. They all wait for an ngcc lockfile to unlock and timeout.